check_limits.py

```diff
@@ -1,18 +1,77 @@
+import re
+# import googletrans
+from googletrans import Translator
```

Comment:
create a requirements.txt with googletrans, so that anyone else using your code knows what to install.
It will also be picked up automatically by the main_workflow.yml